### PR TITLE
Remove network request for pngquant src

### PR DESCRIPTION
**Pull request by kevcenteno**
### Problem
CI servers can sometimes fail to fetch the required bins and/or pngquant src.  

### Solution
1. Put the pngquant src in the vendor directory
2. Use `binBuild.file` instead of `binBuild.src`

### CODE REVIEW

**Comment by kevcenteno** [comment]
@sindresorhus Any thoughts?

**Comment by kevcenteno** [comment]
@sindresorhus Is this still maintained?

**Comment by sindresorhus** [comment]
You committed a change to `vendor/linux/x64/pngquant`. That needs to be removed.

**Comment by sindresorhus** [comment]
I think this is a good idea 👍

**Comment by sindresorhus** [comment]
You also need to add it to the `files` field in package.json

**Comment by kevcenteno** [comment]
@sindresorhus I've made the requested changes.  Thank you for reviewing!

**Comment by chasebolt** [comment]
+1 on this as it would resolve a network issue i am having with this package. PR #83 can be closed out if this PR is merged in. this would also resolve #78.

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
 	"files": [
 		"cli.js",
 		"index.js",
-		"lib"
+		"lib",
+		"vendor/src-pngquant/pngquant-2.10.1-src.tar.gz"
```
Comment (realityking) [inline package.json]: Maybe just add `vendor/src-pngquant`? One less place to change when updating to a newer version.

**Pull request merged**